code/modules/reagents/chemistry/recipes/toxins.dm

```diff
@@ -81,10 +81,10 @@
 	results = list(/datum/reagent/toxin/ghoulpowder = 2)
 	required_reagents = list(/datum/reagent/toxin/zombiepowder = 1, /datum/reagent/medicine/epinephrine = 1)
 
-/datum/chemical_reaction/mindbreaker
-	name = "Mindbreaker Toxin"
-	id = /datum/reagent/toxin/mindbreaker
-	results = list(/datum/reagent/toxin/mindbreaker = 5)
+/datum/chemical_reaction/LSD
+	name = "LSD"
+	id = /datum/reagent/toxin/LSD
+	results = list(/datum/reagent/toxin/LSD = 5)
 	required_reagents = list(/datum/reagent/silicon = 1, /datum/reagent/hydrogen = 1, /datum/reagent/medicine/charcoal = 1)
```

Comment:
Man what a nonsensical recipe for LSD.

Reply:
do you propose a change for this or?

Reply:
you could name it to something that sounds bad but doesn't exist in reality so doesn't need a proper recipe, like mindbreaker

Reply:
Some kind of acid, water, carbon, and nitrogen would be kinda realistic.

Reply:
Or maybe water, carbon, nitrogen and heat, so that the tide still has access to their funny hallucination chem without having to steal sulphuric acid.